pytorch_lightning/utilities/imports.py

```diff
@@ -114,7 +115,7 @@ def _compare_version(package: str, op: Callable, version: str, use_base_version:
 _NEPTUNE_GREATER_EQUAL_0_9 = _NEPTUNE_AVAILABLE and _compare_version("neptune", operator.ge, "0.9.0")
 _OMEGACONF_AVAILABLE = _package_available("omegaconf")
 _POPTORCH_AVAILABLE = _package_available("poptorch")
-_HABANA_FRAMEWORK_AVAILABLE = _package_available("habana_frameworks")
+_PSUTIL_AVAILABLE = _package_available("psutil")
```

Comment:
let's add the min version too? `1.9.0`. Just like `_RICH_AVAILABLE`.

Reply:
Why 1.9.0?